### DISCUSSION TITLE
Who is using react-native-iap?

### DISCUSSION THREAD

**Discussion by hyochan** [hyochan/react-native-iap#1343] [General]
Let's hear more voices in the current thread. Comming from #1247

Sharing your app, your company, or your business will be helpful for other developers to take a look at.

**Comment by ShivamJoker** [comment]
Hey I am using this library in my [SQL-Play](https://sqlplay.net) app which allows you to run sql queries in your phone.
Github link - https://github.com/ShivamJoker/SQL-Play

**Comment by grubstarstar** [comment]
We are about to release a dating app in Australia using this. Still working out some issues with it, but getting there.

**Comment by YaoHuiJi** [comment]
I'm using it for my time management app [iDeliberate](https://apps.apple.com/app/apple-store/id1436043404?pt=879821&ct=github&mt=8), thank you so much for all of your efforts

**Comment by aditya-keri-betterhalf** [comment]
We're using this library in our app - [Betterhalf.ai](https://play.google.com/store/apps/details?id=com.betterhalf&hl=en_IN&gl=US).

**Comment by textual** [comment]
I am looking to implement IAP of Subscriptions within my Expo app.

**Reply by hyochan** [reply]
@textual Nice~ Could you share you app?

**Comment by buonan** [comment]
If you must implement a backend to track refunds, then how are people handling that using react-native-iap or not?  Should I go with revenue cat?

**Reply by hyochan** [reply]
@buonan Sorry this is off topic so I hope you open seperate discussion!

**Comment by hyochan** [comment]
Hey everyone 👋 We just launched a **"Who uses OpenIAP?"** showcase on [openiap.dev](https://www.openiap.dev) — and we'd love to feature the apps shared in this thread!

Since this thread is now read-only, submissions have moved here:

👉 **https://github.com/hyodotdev/openiap/issues/280**

You can also reply to our [announcement on X](https://x.com/hyodotdev/status/2084648736119312497) or email **hyo@hyo.dev** with:

- App name + one-line description
- Square logo (PNG/SVG)
- App Store / Google Play links
- Which library you use (react-native-iap, expo-iap, …)

Apps are featured only with your permission — you can request changes or removal anytime. 🙏
